.changeset/shaggy-melons-drive.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/catalog-model': patch
```

Comment:
Should this be minor? In the sense that it's "a new feature".

Reply:
I'll update to minor